blackbox/docs/general/information-schema.txt

```diff
@@ -137,7 +137,7 @@ columns::
 +----------------------------------+------------------------------------------------------------------------------------+-------------+
 | ``reference_generation``         | Specifies how values in the self-referencing column are generated                  | ``String``  |
 +----------------------------------+------------------------------------------------------------------------------------+-------------+
-| ``routing_hash_function``        | The hash function used for internal routing                                        | ``String``  |
+| ``routing_hash_function``        | The name of the hash function used for internal routing.                           | ``String``  |
```

Comment:
we don't end the other sentences here with `.`, so fore consistency please remove